website/src/content/blog/biome-v1-6.mdx

```diff
@@ -0,0 +1,237 @@
+
+Biome now accepts the file `biome.jsonc` as configuration! Now, you can insert all the comments you want in a configuration.
+
+### `extends` resolves dependencies
+
+From this version, Biome can the `extends` property can *resolve* other configuration files that are installed and exported as a dependency.
+
+There are few important steps in order to make the configuration discoverable. The file must be exported from a `"module"` package, and the file should be exported in your `package.json` like this:
+
+```json name="shared-configs/package.json"
+{
+  "name": "@shared-configs",
+  "type": "module",
+  "exports": {
+    "./biome": "./biome.json"
+  }
+}
+```
+
+This set up allows to expose a specifier `@shared-configs/biome`, which you use inside your `biome.json` file.
+
+```json name="project/biome.json"
+{
+  "extends": ["@shared-configs/biome"]
+}
+```
+
+The resolution of the dependencies is powered by the library [`oxc-resolver`](https://github.com/oxc-project/oxc-resolver), one of the many libraries provided by the [OXC project](https://oxc-project.github.io/). It's battle-tested and spec compliant!
+
+:::note
+You can also export `biome.jsonc` files!
+
+```json name="shared-configs/package.json" ins="./biome.jsonc"
+{
+  "name": "@shared-configs",
+  "type": "module",
+  "exports": {
+    "./biome": "./biome.jsonc"
+  }
+}
+```
+:::
+
+### Reduced memory footprint
+
+We **reduced** the size our configuration **by a factor of 6.5**! This change might not have massive effects on the speed of the program, but it greatly reduced the memory used when running the CLI or the LSP.
+
+## New formatting options
+
+Other than fixes, the formatter provides two new options that should improve the compatibility with Prettier parity
+
+### Option `attributePosition`
+
+The contributor [@octoshikari](https://github.com/octoshikari) implemented this new feature by themselves! Huge thank you for helping the Biome project.
+
+When `formatter.attributePosition` has the value `multiline`, all attributes of HTML-ish languages (JSX/TSX as for time of writing) will be collapsed on multiple lines regardless of their numbers:
+
+
+<div class="card">
+    <div class="card-column">
+        <h4>With variant `auto` (default)</h4>
+        The attributes are automatically formatted, and they will collapse in multiple lines only when they hit certain criteria.
+
+        ```jsx title="file.jsx"
+        <Button as="link" style="primary" href="https://example.com">
+            Hit me
+        </Button>
+        ```
+    </div>
+    <div class="card-column">
+        <h4>With variant `multiline`</h4>
+        The attributes are always formatted on multiple lines, regardless.
+
+        ```jsx title="file.jsx"
+        <Button
+            as="link"
+            style="primary"
+            href="https://example.com"
+        >
+            Hit me
+        </Button>
+        ```
+    </div>
+</div>
+
+
+### Option `json.formatter.trailingComma`
```

Comment:
@ematipico Please forgive me for being a little paranoid about the plural trailing `s`.

I just double checked the latest nightly release. We should use `trailingCommas` everywhere for the `json` formatter, and fix the option choices in CLI: `--json-formatter-trailing-commas=<omit|allow>`, and we might want to explain why it is `trailingComma` for the `js` formatter but decide to use `trailingCommas` for the `json` formatter (to keep consistency with other options).

And maybe we should mark the old option name as deprecated and implement and recommend users to use the new option name so that we can completely remove it in the future with less friction.

Reply:
Ah shoot, I forgot to follow up 😆  you're right

Reply:
No worries. Thank you for putting up with me :D